react/I18n/index.jsx

```diff
@@ -20,9 +20,16 @@ export class I18n extends Component {
   }
 
   init(props) {
-    const { lang, dictRequire, context, defaultLang } = props
-
-    this.translation = initTranslation(lang, dictRequire, context, defaultLang)
+    const {
+      polyglot,
+      lang,
+      dictRequire,
+      context,
+      defaultLang = DEFAULT_LANG
```

Comment:
Shouldn't the DEFAULT_LANG be configured as a defaultProp ?

Reply:
Hmmm, not sure what we'd get by configuring it as a defaultProp instead of using a default value to be honest, what do you think?

Reply:
Other methods would have access to it.

Reply:
Good point :+1: